packages/app/src/App.vue

```diff
@@ -5,10 +5,30 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { defineComponent, watch } from 'vue';
+import { useRouter, useRoute } from 'vue-router';
+
+import { isAuthenticated } from './compositions/useAuthentication';
 
 export default defineComponent({
   name: 'App',
+
+  setup() {
+    const route = useRoute();
+    const router = useRouter();
+
+    watch(
```

Comment:
I understand that the watch function gets two arguments (object to watch, callback to call on change).
Why is your first argument an arrowfunction?

Reply:
@julianhahn Dunno what the difference is between watching with an arrow-function or directly watching the ref. https://v3.vuejs.org/api/computed-watch-api.html#watching-a-single-source

Could you maybe try it with the ref directly?